packages/bootstrap/README.md

```diff
@@ -0,0 +1,25 @@
+# @kaizen/bootstrap
+
+Bootstrap is a place for Kaizen-specific styles that cannot be contained within a component package (such as Button).
+
+Note that this package is _not_ for browser normalisations or polyfills. Instead refer to @cultureamp/frontend-runtime:
+https://github.com/cultureamp/frontend-ops/blob/master/packages/frontend-runtime/lib/index.ts
+
+This package exports two assets; a Sass file and a React component. For most use cases it is easiest to wrap your App in the React component: 
+
+```
+// App.tsx
+import Bootstrap from "@kaizen/boostrap"; 
+
+const YourApp = () => (
+  <Bootstrap>
+    /* your React app goes here */
+  </Bootstrap>
+);
+```
+
+If for whatever reason this is not possible, you can import the Sass file which will give you all the necessary @font-face definitions: 
+```
+// index.module.sass
+@import "~@kaizen/boostrap/fonts.module.css" 
```

Comment:
If it's not possible for a user to use the React wrapper (for whatever reason), should there be an alternate file that adds these definitions to the `body` so that a consumer just needs to import one Sass file? Something like...

```
// index.module.scss
@import "~@kaizen/boostrap/fonts.module.css"

body {
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  background-color: $kz-color-stone;
}
```

Reply:
Nope, bad idea. This flexibility is not required.